data/api/server-server/joins-v1.yaml

```diff
@@ -218,6 +218,7 @@ paths:
 
   "/send_join/{roomId}/{eventId}":
     put:
+      deprecated: true
```

Comment:
We should also hit the `send_leave` endpoint with this, per the MSC mentioned in https://github.com/matrix-org/matrix-spec/pull/1518#issuecomment-1544322312